org.lflang.tests/src/org/lflang/tests/compiler/FormattingUnitTests.java

```diff
@@ -56,16 +65,34 @@ public void testState() {
                 reactor Destination {
                     state  one_init: tag_t( {= NEVER_TAG_INITIALIZER =})
                     state no_init:   tag_t
-                     state list_init(1,2)
+                     state list_init(1,2) // this syntax is deprecated
```

Comment:
Why not change this to the new syntax?
```suggestion
                     state list_init = (1,2)
```

Reply:
I want to make sure that the formatter does not rewrite this syntax, even if it is deprecated. The validator emits a warning but the update here should be manual. `(1, 2)` is also not yet a valid expression in the python target